metadata-io/src/main/java/com/linkedin/metadata/entity/ebean/EbeanAspectDao.java

```diff
@@ -498,15 +498,19 @@ public Map<String, Long> getNextVersions(@Nonnull final String urn, @Nonnull fin
     if (exp == null) {
       return result;
     }
+    // Ensure Ordering of Aspect version
+    exp.orderBy("version");
```

Comment:
Instead of adding an order by to the ebeans request. Can we do groupby on java side? 

dbResults.stream().collect(Collectors.toMap(EbeanAspectV2.PrimaryKey::getAspect, Function.identity(),
        BinaryOperator.maxBy(Comparator.comparing(EbeanAspectV2.PrimaryKey::getVersion)))).foreach(key => result.put(aspectName, nextVal))

Reply:
Whats the benefit here dexter?

Reply:
I'm okay with either, for the record

Reply:
Are we not able to add a groupBy to the SQL side?

Reply:
no:(

Reply:
Time to move to Hibernate? :P

Reply:
After discussion, I think the best way forward is to make sure the ebeans query returns the correct result (one row per urn, aspect pair). To unblock @arunvasudevan for now, I am ok with shipping this fix as is.

Reply:
Awesome, Thanks guys!